src/pages/drive/addIns/driveAddIns.tsx

```diff
@@ -0,0 +1,529 @@
+import { ReactNode } from "react";
+import { IconChevronRightSmall } from "@geotab/react-component-library";
+import Accordion from "../../../components/Accordion/Accordion";
+import CodeSample from "../../../components/CodeSamplesContainer/CodeSample";
+import InformationalBox from "../../../components/InformationalBox/InformationalBox";
+import { Link } from "react-router-dom";
+import driveAddInImage from "./../../../assets/images/driveAddIns/drive-addins_0.png";
+
+const overview: ReactNode = <div className="paragraph">
+    <p>The MyGeotab Add-In structure can be applied to the Geotab Drive application as well, providing you the ability to extend the functionality for drivers in an environment with sensors (e.g. geolocation and acceleration) and actuators (e.g. voice alerts and notifications). This environment must also be completely operable in an offline state — so your Add-In must be able to handle having no internet connection, or sleeping in the background of the mobile device.</p>
+    <p>Please read the <Link to="/myGeotab/addIns/developingAddIns">Developing Add-Ins</Link> guide first before attempting a Geotab Drive Add-In.</p>
+    <p>All Add-Ins that have been designed to work with MyGeotab will work on the Geotab Drive app as well. Your Add-In will be completely downloaded for all referenced links, images, and scripts upon user login. This way, as the user is authenticated over the internet — they will have your Add-In with them as they travel or disconnect from the network. If your Add-In requires dynamic loading of CSS, images, or JavaScript — these requests will fail if the user does not have a network connection. As such you should either: include all dependencies on creation of the Add-In, explicitly link to them, or provide a fallback if <strong>state.online</strong> returns False. Geotab Drive Add-Ins will also display differently, on both the dashboard and the menu.</p>
+    <img src={driveAddInImage} alt="Drive dashboard" />
+    <p>To make an Add-In on the Geotab Drive app, the <strong>item</strong> in your configuration file must have a <strong>path</strong> that equals "<strong>DriveAppLink/</strong>" (including a trailing forward slash). The menuName, url, and version will remain unchanged. For example:</p>
+    <h2>Listing 1 — Geotab Drive "Item" Configuration</h2>
+    <CodeSample
+        language="json"
+        code={`"items": [{
+    "version": "1.0",
+    "url": "addinFile.html",
+    "path": "DriveAppLink/",
+    "menuName": {
+        "en": "English Label",
+        "fr": "French Label"
+    }
+}]`}></CodeSample >
```

Comment:
```suggestion
    <CodeSample
        language="json"
        code={`"items": [{
    "version": "1.0",
    "url": "addinFile.html",
    "path": "DriveAppLink/",
    "menuName": {
        "en": "English Label",
        "fr": "French Label"
    }
}]`}/>
```
This can be self closing cause it has no children inside of it.